### PR TITLE
Issue91 button implementation

### PR DESCRIPTION
**Pull request by yoouung**
- working on `upload`, `filelist`, `rules`, `report` pages
- add buttons on each pages
- modify `Router` to implement _nested router_ and _outlet_

### CODE REVIEW

**Review by doorooful — CHANGES_REQUESTED** [review]
Could you modify commit message?

File: dt-demo-fe/src/pages/demo/upload.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+// import UploadBtn from "../../components/File/uploadBtn";
```
Comment (doorooful) [inline dt-demo-fe/src/pages/demo/upload.js:2]: Can you remove unnecessary line?

File: dt-demo-fe/src/Router.js [hunk 2]
```diff
@@ -19,6 +23,13 @@ const Router = () => {
                 <Route path="/filter" element={<Filter />} />
                 <Route path="/converter" element={<Converter />} />
                 <Route path="/statistics" element={<Statistics />} />
+            </Routes> */}
+            <Routes>
+                <Route path="/" element={<Navigate replace to="/upload" />} />
```
Comment (doorooful) [inline dt-demo-fe/src/Router.js:28]: Can you move these routes under `/filter`?
i.e. `/filter/upload`, `/filter/filelist`

File: dt-demo-fe/src/pages/demo/upload.js (hunk 1 above)
Comment (yoouung) [reply dt-demo-fe/src/pages/demo/upload.js]: checked!

File: dt-demo-fe/src/Router.js (hunk 2 above)
Comment (yoouung) [reply dt-demo-fe/src/Router.js:28]: checked!
I added __Outlet__ component in `02_filter.js`

**Comment by yoouung** [comment]
# /filter
image

# /filter/filelist
image

# /filter/rules
image

# /filter/report
image

**Comment by doorooful** [comment]
Thanks for your effort☺️

**Review by doorooful — APPROVED** [review]

**Pull request merged**